src/main/java/pw/itr0/kaba/exception/MissImplementationException.java

```diff
@@ -0,0 +1,57 @@
+package pw.itr0.kaba.exception;
+
+/**
+ * The {@code MissImplementationException} class is a generic exception class to wrap checked Exception which cannot occur.
+ *
+ * @author ryotan
+ * @since 1.0.0
+ */
+public class MissImplementationException extends RuntimeException {
+
+    /**
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [First sentence of Javadoc is incomplete (period is missing) or not present.](https://www.codacy.com/app/ryotan/kaba-vault/file/2578430310/issues/source?bid=3298589&fileBranchId=3301306#l11)